packages/apollo-datasource-rest/README.md

```diff
@@ -41,6 +41,80 @@ class MoviesAPI extends RESTDataSource {
 }
 ```
 
+## API Reference
+View the source code to see the all the properties and functions that can be overridden and their specific parameters. This section lists the usage and default behavior of the `RESTDatasource` class.
+
+### Constructor Parameters
+#### `httpFetch`
+
+Optional constructor option which allows overriding the `fetch` implementation used when calling data sources.
+
+### Properties
+#### `baseURL`
+Optional value to use for all the REST calls. If it is set in your class implementation, this base URL is used as the prefix for all calls. If it is not set, then the value passed to the REST call is exactly the value used.
+
+```js title="baseURL.js"
+class MoviesAPI extends RESTDataSource {
+  constructor() {
+    super();
+    this.baseURL = 'https://movies-api.example.com/';
+  }
+
+  // GET
+  async getMovie(id) {
+    return this.get(
+      `movies/${encodeURIComponent(id)}` // path
+    );
+  }
+}
+```
+
+#### `requestCacheEnabled`
+By default, `RESTDataSource` caches all outgoing GET **requests** in a separate memoized cache from the regular response cache. It does this to prevent duplicate calls that might happen in rapid succession.
+If a request is made with the same cache key (URL by default) with an HTTP method other than GET, the cached request is then cleared.
+
+If you would like to disable the GET request cache, set the `requestCacheEnabled` property to `false`.
```

Comment:
Can we explain here why this might be desirable? I'm actually unsure, so I imagine others reading this would also want to know what problem this is solving.